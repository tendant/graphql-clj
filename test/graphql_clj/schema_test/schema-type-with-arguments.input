type Arguments {
  multipleReqs(x: Int!, y: Int!): Int!
  booleanArgField(booleanArg: Boolean): Boolean
  floatArgField(floatArg: Float): Float
  intArgField(intArg: Int): Int
  nonNullBooleanArgField(nonNullBooleanArg: Boolean!): Boolean!
  booleanListArgField(booleanListArg: [Boolean]!): [Boolean]
}
