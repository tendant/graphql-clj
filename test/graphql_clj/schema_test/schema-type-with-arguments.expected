{:objects
 {:Arguments
  {:fields
   {:multipleReqs
    {:args {:x {:type (non-null Int)}, :y {:type (non-null Int)}},
     :type (non-null Int)},
    :booleanArgField
    {:args {:booleanArg {:type Boolean}}, :type Boolean},
    :floatArgField {:args {:floatArg {:type Float}}, :type Float},
    :intArgField {:args {:intArg {:type Int}}, :type Int},
    :nonNullBooleanArgField
    {:args {:nonNullBooleanArg {:type (non-null Boolean)}},
     :type (non-null Boolean)},
    :booleanListArgField
    {:args {:booleanListArg {:type (non-null (list Boolean))}},
     :type (list Boolean)}}}}}
